tests/System.IO.Compression.Tests/BrotliPrimitivesTests.cs

```diff
@@ -10,6 +10,125 @@ public class BrotliTests
     {
         static string brTestFile(string fileName) => Path.Combine("BrotliTestData", fileName);
 
+        [Fact(Skip = "Fails in VS - System.BadImageFormatException : An attempt was made to load a program with an incorrect format.")]
```

Comment:
Any ideas how to resolve these test failures in VS? We should enable these tests if possible.